### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by SebastiaanNijland**
* Set `type="button"` with a `defaultProp` instead of logic
* Use Button components instead of plain buttons
* Added a simple test for the Accordion component
* Added a small test case for a Label component with multiple children
* Removed some already implemented TODO’s

### CODE REVIEW

File: src/CloseButton.js [hunk 1]
```diff
@@ -10,14 +12,13 @@ class CloseButton extends React.Component {
   render() {
     const { label, onClick } = this.props;
     return (
-      <button
-        type="button"
+      <Button
```
Comment (jquense) [inline src/CloseButton.js]: This won't work. Close buttons aren't implemented in terms of Buttons they are a separate thing. It's odd I know but that's how the styling works

File: src/NavbarToggle.js [hunk 2]
```diff
@@ -39,19 +39,19 @@ class NavbarToggle extends React.Component {
 
     if (children) {
       return (
-        <button {...buttonProps}>
+        <Button {...buttonProps}>
           {children}
-        </button>
+        </Button>
```
Comment (jquense) [inline src/NavbarToggle.js]: Same as close button, this isn't a bootstrap btn

File: src/NavbarToggle.js [hunk 3]
```diff
@@ -39,19 +39,19 @@ class NavbarToggle extends React.Component {
...
 
     return (
-      <button {...buttonProps}>
+      <Button {...buttonProps}>
         <span className="sr-only">Toggle navigation</span>
         <span className="icon-bar" />
         <span className="icon-bar" />
         <span className="icon-bar" />
-      </button>
+      </Button>
```
Comment (jquense) [inline src/NavbarToggle.js]: Ditto

File: test/AccordionSpec.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    let instance = ReactTestUtils.renderIntoDocument(
+      <Accordion>
+        Accordion content
+      </Accordion>
+    );
+
+    const accordion = ReactDOM.findDOMNode(instance);
+
+    assert.equal(accordion.nodeName, 'DIV');
+    assert.equal(accordion.getAttribute('role'), 'tablist');
```
Comment (jquense) [inline test/AccordionSpec.js]: Can this go in the PanelGroup tests?

Comment (SebastiaanNijland) [inline test/AccordionSpec.js]: Of course!

File: src/CloseButton.js (hunk 1 above)
Comment (SebastiaanNijland) [inline src/CloseButton.js]: Ahh, too bad. But makes sense.

File: src/Button.js [hunk 5]
```diff
@@ -47,7 +48,7 @@ class Button extends React.Component {
     return (
       <Component
         {...elementProps}
-        type={elementProps.type || 'button'}
```
Comment (taion) [inline src/Button.js]: slightly different semantically... do we ever pass in falsy (but defined) `type` here?

Comment (SebastiaanNijland) [inline src/Button.js]: That shouldn't be possible though, since type is defined as `oneOf` the truthy button types above?

Comment (taion) [inline src/Button.js]: ah, okay, great

**Comment by taion** [comment]
What's up?

**Comment by SebastiaanNijland** [comment]
The PR is quite out of date :) I'll hopefully be able to get it up to date soon.

**Pull request closed without merging**